Index.html

```diff
@@ -1,179 +1,283 @@
+            <li class="header__navication"><a class="nav__desktoplink" href="#headline">Hello</a></li>
+            <li class="header__navication"><a class="nav__desktoplink" href="#works">Works</a></li>
+            <li class="header__navication"><a class="nav__desktoplink" href="#about">About</a></li>
+            <li class="header__navication"><a class="nav__desktoplink" href="#contact">Contact</a></li>
         </ul>
-      </li>
-      <li>Skills
-        <ul id="skills">
-          <li>Postgres</li>
-          <li>RabbitMQ</li>
-          <li>GitHub</li>
-          <li>Linux</li>
-          <li>Docker</li>
+
+        <img class="mobile__photos mobile__invisible nav__bottomImage" src="./Contents/Resources/Pictures/bottom_menu_open.svg" alt="">
+    </header>
+
+    <main class="main">
+        <section id="headline" class="headline">
+            <article class="text__article">
+                <h1 class="title">
+                    Mohammad Mehdi <br>
+                    Pazouki
+                </h1>
+                <p class="bio">
+                    Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don't hesitate and contact me.
+                </p>
+            </article>
+
+            <button class="buttons btn">
+                Start Collaboration
+            </button>
+
+            <img src="./Contents/Resources/Icons/dots.svg" alt="" class="hello__dots">
+            <img id="desktop__geometry" src="./Contents/Resources/Pictures/helloGeometry.svg" alt="">
+            <img id="geometry" src="./Contents/Resources/Pictures/image_geometry_2.svg" alt="">
+        </section>
+
+        <section id="works" class="works">
+            <h2 class="h2 portfolio__hearder">Projects</h2>
+            <article class="portfolio project__one">
+                <img src="./Contents/Resources/Pictures/Correspondence_img.jpg" alt="project one is a web site" class="project__photo project__onePhoto">
+                <div class="project__info hover__one">
+                    <h3 class="project__title">Correspondence</h3>
+                    <ul class="project__stack__container">
+                        <li class="project__stack stack1 one__stacks">HTML/CSS</li>
+                        <li class="project__stack stack3 one__stacks">JavaScript</li>
+                        <li class="project__stack stack3 one__stacks">SQL</li>
+                    </ul>
+                    <div class="project__button__holder">
+                        <button class="buttons project__button">
+                            See this project
+                            <span class="icon arrow material-symbols-outlined">
+                                arrow_right_alt
+                            </span>
+                        </button>
+                    </div>
+                </div>
+            </article>
```

Comment:
According to the Figma template, you should Follow the Design to make your project exactly like the Design, Kindly I suggest you follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=X7Yh3FuNhyLhpYDE-0) to make your project match the design template.

> ### The Figma Design
>
><img width="182" alt="Screenshot 2023-04-28 at 15 24 05" src="https://user-images.githubusercontent.com/91237525/235236095-8184ee77-2593-4e18-8d3b-324d08907f48.png">
>
>
>### Your Design
>
><img width="282" alt="Screenshot 2023-04-28 at 15 24 24" src="https://user-images.githubusercontent.com/91237525/235236202-a80cf124-526e-47d7-8f9a-f49939d928eb.png">

Kindly, I suggest you use the font-size, letter-spacing, as it is in the Figma template.